src/Operations.jl

```diff
@@ -255,12 +255,12 @@ is_tracking_registry(pkg) = !is_tracking_path(pkg) && !is_tracking_repo(pkg)
 isfixed(pkg) = !is_tracking_registry(pkg) || pkg.pinned
 
 function collect_developed!(ctx::Context, pkg::PackageSpec, developed::Vector{PackageSpec})
-    source = project_rel_path(ctx, source_path(pkg))
+    source = project_rel_path(ctx, source_path(ctx, pkg))
     source_ctx = Context(env = EnvCache(projectfile_path(source)))
     pkgs = load_all_deps(source_ctx)
     for pkg in filter(is_tracking_path, pkgs)
         # normalize path
-        pkg.path = Types.relative_project_path(ctx, project_rel_path(source_ctx, source_path(pkg)))
+        pkg.path = Types.relative_project_path(ctx, project_rel_path(source_ctx, source_path(ctx, pkg)))
```

Comment:
it should be `source_path(source_ctx, pkgs)` here

Reply:
Thanks!